### PR TITLE
Feature/abstract images event from regular event

### PR DESCRIPTION
**Pull request by mrcthms**
Fixes a typo with the `trigger` method, documented the code, and refactored to make the code a bit more readable.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,13 +8,40 @@
 
   var $window = $(window);
 
+  /**
+   * Fire a custom event on $scriptTag, to say that the noscript content has
+   * been switched out.
+   * @param  {HTMLElement} $scriptTag  The <noscript>/<script> tag which houses the hidden content
```
Comment (callumacrae) [inline index.js]: `HTMLElement` -> `jQuery`

Comment (mrcthms) [inline index.js]: @callumacrae updated 👍

**Comment by callumacrae** [comment]
Is this just refactoring or does it need a release?

**Comment by mrcthms** [comment]
@callumacrae Needs a patch release, fixes a typo.

**Comment by callumacrae** [comment]
Done, released as 0.2.3.

I got the following warning when running the tests:

```
Error loading resource https://images.contentful.com/apgig5q1psgn/6UE4dNSawoc4i8YkaM8MyA/14e1d46aca8c01c703328ec5d1d265c4/shipping-60x60.png (6). Details: SSL handshake failed
```

This didn't cause test failure, though—should it? Also, would it be better to use a simple image from imgur to avoid issues like this and it possibly being deleted in the future?

**Comment by mrcthms** [comment]
@callumacrae Thanks!  And regarding the broken image, it probably should, yeah. Will write a test for that.

**Comment by callumacrae** [comment]
awesome :)

**Pull request merged**